SPECS/python-docutils/python-docutils.spec

```diff
@@ -51,21 +50,22 @@ python3 setup.py build
 popd
 
 %install
+
 pushd ../p3dir
 python3 setup.py install --prefix=%{_prefix} --root=%{buildroot}
-mv %{buildroot}/%{_bindir}/rstpep2html.py %{buildroot}/%{_bindir}/rstpep2html3.py
-mv %{buildroot}/%{_bindir}/rst2xml.py %{buildroot}/%{_bindir}/rst2xml3.py
-mv %{buildroot}/%{_bindir}/rst2xetex.py %{buildroot}/%{_bindir}/rst2xetex3.py
-mv %{buildroot}/%{_bindir}/rst2s5.py %{buildroot}/%{_bindir}/rst2s53.py
-mv %{buildroot}/%{_bindir}/rst2pseudoxml.py %{buildroot}/%{_bindir}/rst2pseudoxml3.py
-mv %{buildroot}/%{_bindir}/rst2odt_prepstyles.py %{buildroot}/%{_bindir}/rst2odt_prepstyles3.py
-mv %{buildroot}/%{_bindir}/rst2odt.py %{buildroot}/%{_bindir}/rst2odt3.py
-mv %{buildroot}/%{_bindir}/rst2man.py %{buildroot}/%{_bindir}/rst2man3.py
-mv %{buildroot}/%{_bindir}/rst2latex.py %{buildroot}/%{_bindir}/rst2latex3.py
-mv %{buildroot}/%{_bindir}/rst2html5.py %{buildroot}/%{_bindir}/rst2html53.py
-mv %{buildroot}/%{_bindir}/rst2html.py %{buildroot}/%{_bindir}/rst2html3.py
+for BINARY in rstpep2html rst2xml rst2xetex rst2s5 rst2pseudoxml rst2odt_prepstyles rst2odt rst2man rst2latex rst2html5 rst2html rst2html4
+do
+  mv %{buildroot}/%{_bindir}/$(echo $BINARY).py %{buildroot}%{_bindir}/$(echo $BINARY)3.py
```

Comment:
>$(echo $BINARY) [](start = 29, length = 15)

Why `$(echo $BINARY)` instead of only `$BINARY`? I'd suggest simplifying this here and elsewhere in this spec.

Reply:
I checked with just putting '%{_bindir}/$BINARY.py'. It gets interpreted as 'usr/bin/.py'. So, I am leaving it as '%{_bindir}/$(echo $BINARY).py'